i18n/locales/en/translations.ts

```diff
@@ -67,69 +67,71 @@ export const translations = {
   },
 
   reaping_rewards: {
-    title: 'Of blocks and miners',
-    subtitle:
-      '"To understand the mystery and magic, you must start at the beginning…"',
-    paragraph_one: `Satoshi, the pseudonymous inventor of bitcoin, also mined the first bitcoin block. He left you a hidden clue in the very first bitcoin transaction that was ever made.`,
-    paragraph_two: `Each block in the bitcoin blockchain contains a set of transactions. A nice way to view these blocks and the things they contain is by using a block explorer.`,
+    title: null,
```

Comment:
With the new changes to `Introduction.ts` We should no longer need to have this translation key